src/providers/MeetingProvider/index.tsx

```diff
@@ -59,6 +59,10 @@ interface Props {
    * The [ActiveSpeakerPolicy](https://aws.github.io/amazon-chime-sdk-js/interfaces/activespeakerpolicy.html) object that you want to use in the meeting session.
    */
   activeSpeakerPolicy?: ActiveSpeakerPolicy;
+  /**
+   * Maximum amount of time in milliseconds to allow for reconnecting. Default is 120 seconds. (https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#reconnecttimeoutms)
```

Comment:
I am not sure how this looks in storybook, but, like above can you please do this as a markdown link?

```
Maximum amount of time in milliseconds to allow for reconnecting. Default is [120 seconds](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#reconnecttimeoutms).
```